.travis.yml

```diff
@@ -70,7 +70,7 @@ install:
         sudo apt-get -y install libgstreamer1.0-dev gstreamer1.0-alsa gstreamer1.0-plugins-base;
         sudo apt-get -y install python-dev libsmpeg-dev libswscale-dev libavformat-dev libavcodec-dev libjpeg-dev libtiff4-dev libX11-dev libmtdev-dev;
         sudo apt-get -y install python-setuptools build-essential libgl1-mesa-dev libgles2-mesa-dev;
-        sudo apt-get -y install xvfb pulseaudio;
+        sudo apt-get -y install xvfb pulseaudio xsel;
```

Comment:
Here I use `xsel` but we can also use `xclip`. There is no special reason to use `xsel`, I used that because is the package that I use in my development machine...
should we use `xclip`?

Reply:
I forgot, we also could drop these changes, I know that the test is ok without installing anything more...so actually we don't need that because does not solve any travis error, but I think that makes easier to find the real problem...
Here a test without `xsel` (scroll to end):
  - https://travis-ci.org/kivy/kivy/jobs/490796639

And here one with `xsel` (scroll to end):
  - https://travis-ci.org/kivy/kivy/jobs/490991404